apps/api/src/server/build-app.js

```diff
@@ -24,6 +24,17 @@ const buildApp = (
 		addSwaggerUi(app);
 	}
 
+	app.get('/health', async (req, res) => {
+		try {
+			await databaseConnector.$queryRaw`SELECT 1;`;
+		} catch {
+			res.status(500).send('Database connection failed during health check');
+			return;
+		}
+
+		res.status(200).send('OK');
+	});
```

Comment:
moved health-checks to not require authorisation

Reply:
Good plan. Probably worth a comment in code so people know the order/position is important.